dependencywalker.txt

```diff
@@ -1,16 +1,19 @@
 [Section]
 Name = Dependency Walker
-Description = A free utility that scans any 32-bit or 64-bit Windows module (exe, dll, ocx, sys, etc.) and builds a hierarchical tree diagram of all dependent modules.
+Description = MsDepends is a free utility that scans any 32-bit or 64-bit Windows module (exe, dll, ocx, sys, etc.) and builds a hierarchical tree diagram of all dependent modules.
 LicenseType = 2
 Category = 7
 URLSite = https://dependencywalker.com
 Version = 2.2
 URLDownload = https://dependencywalker.com/depends22_x86.zip
-SizeBytes = 468618
+SHA1 = bfec714057e8449b0246051be99ba46a7760bab9
+SizeBytes = 610769
 Installer = Generate
 
 [Section.amd64]
 URLDownload = https://dependencywalker.com/depends22_x64.zip
+SHA1 = 4831d2a8376d64110ff9cd18799fe6c69509d3ea
+SizeBytes = 468618
```

Comment:
A bit surprising that the x64 has smaller filesize than the x86. That is pretty uncommon. No biggy, I am just expressing my mind here in case you swapped the numbers accidentally. If not, then fine.

Reply:
Yes, it is correct. The 64-bit depends.exe is a lot smaller, perhaps missing Windows 9x code? depends.dll is larger.